active/Cross Site WebSocket Hijacking.js

```diff
@@ -0,0 +1,100 @@
+/**
```

Comment:
This mixing of copyright/license with other information is confusing. Maybe split into two comments?

Reply:
Ok, I reordered license and description.

Reply:
I think his point was more that they should be completely separate blocks.

Reply:
@thc202 is this comment issue blocking this pull request? I reordered comment text, so the copy and license is not mixed with script's description. Of course I can split it into two separated comment blocks if it makes a difference.

Reply:
@thc202 ^ ?

Reply:
It would be better to split them, easier to read.